xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEStream.cpp

```diff
@@ -716,7 +718,7 @@ bool CActiveAEStreamBuffers::IsDrained()
 
 void CActiveAEStreamBuffers::SetRR(double rr)
 {
-  if (rr < 1.02 && rr > 0.98)
+  if (fabs(rr - 1.0) < m_atempoThreshold)
```

Comment:
if we passed threshold as parameter of SetRR instead of constructor, users would be able to change threshold on the fly. just add the setting to CActiveAE::OnSettingsChange